src/vg/civcraft/mc/civchat2/listeners/BetterShardsListener.java

```diff
@@ -26,12 +26,15 @@ public BetterShardsListener(CivChat2 plugin) {
 	public void PlayerChangeServer(PlayerChangeServerEvent event){
 		String playerName = NameAPI.getCurrentName(event.getPlayerUUID());
 		UUID playerToReplyUUID = chatMan.getPlayerReply(playerName);
-		if (playerToReplyUUID == null) {
-			return;
+		if (playerToReplyUUID != null) {
+			String playerToReply = NameAPI.getCurrentName(playerToReplyUUID);
+			if(playerToReply != null){
+				MercuryAPI.sendMessage(event.getServerTravelingTo(), "channel" + sep + "reply" + sep + playerName + sep + playerToReply, "civchat2");
```

Comment:
Isnt this the thing handling if you are in a chatchannel? The thing when you just run /tell blackxnt for example